nilearn/externals/README.md

```diff
@@ -3,3 +3,6 @@ This directory contains bundled external dependencies.
 Note for distribution packagers: if you want to remove the duplicated
 code and depend on a packaged version, we suggest that you simply do a
 symbolic link in this directory.
+
+
+Last knwon documentation: https://pyrocore.readthedocs.io/en/latest/tempita.html
```

Comment:
probably better to link it in our official documentation in the maintenance section

Reply:
Actually having a mini issue for a mini PR to add those 2 links to the doc could be a nice good first issue for someone who has never done a PR ever: maybe for the hackathon @man-shu ?